libs/coin-modules/coin-stellar/src/api/index.ts

```diff
@@ -31,8 +31,11 @@ export function createApi(config: StellarConfig): Api<StellarToken> {
   };
 }
 
-async function craft(transactionIntent: TransactionIntent<StellarToken>): Promise<string> {
-  const fees = await estimateFees();
+async function craft(
+  transactionIntent: TransactionIntent<StellarToken>,
+  customFees?: bigint,
+): Promise<string> {
+  const estimatedFees = customFees ? customFees : await estimateFees();
```

Comment:
careful, this check doesn't work for 0 (atleast for numbers, not sure in bigint case)

can we check if customFees != null instead ? (or !==undefined) to atleast be explicit